dist/logic.js

```diff
@@ -0,0 +1,56 @@
+
+class WeatherAPP {
+    constructor(
+    ) {
+        this.apiManger = new ApiManager
+        this.data = []
+        this.searchCity = {}
+        this.coords = {}
+    }
+
+    tempAddToFav(city) {
```

Comment:
maybe think about the names of your functions a bit more. THey can be shorter and better describe what they are doing.
`favoriteCity()`
`getCity()`
`deleteCity()`